src/public/parts/shells/chat/public/stickers/index.html

```
@@ -22,35 +22,35 @@
+				<button id="sticker-upload-button" class="btn btn-secondary" data-i18n="chat.stickers.upload"></button>
+			</div>
 		</div>
-	</div>
 
-	<div class="container mx-auto p-6">
-		<div class="mb-6">
-			<input type="text" id="search-input" class="input input-bordered w-full max-w-md" data-i18n="chat.stickers.search" />
-		</div>
+		<div class="container mx-auto p-6">
+			<div class="mb-6">
+				<input type="text" id="search-input" class="input input-bordered w-full max-w-md" data-i18n="chat.stickers.search" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)